app/code/Magento/InventoryApi/Test/Api/StockRepository/DeleteTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryApi\Test\Api\StockRepository;
+
+use Magento\Framework\Webapi\Rest\Request;
+use Magento\InventoryApi\Api\Data\StockInterface;
+use Magento\TestFramework\Assert\AssertArrayContains;
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class DeleteTest extends WebapiAbstract
+{
+    /**#@+
+     * Service constants
+     */
+    const RESOURCE_PATH = '/V1/inventory/stock';
+    const SERVICE_NAME = 'inventoryApiStockRepositoryV1';
+    /**#@-*/
+
+    /**
+     * @magentoApiDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/stocks.php
+     */
+    public function testDeleteById()
+    {
+        $stockIdToDelete = 10;
+        $expectedStocksAfterDeleting = [
```

Comment:
It will be better to check that after deleting we could not load deleted Stock

For example `StockInterface::NAME => 'Default Stock'` is related to another module
So this way has introduced broken modularity

Module Inventory has base implementation of Inventory business logic
This logic does not depends on concrete Catalog implementation

Сoncept of Default Stock is related to integration Inventory into current Catalog
And we have special module for this integration InventoryCatalog
So need to move this class (and declaration) in InventoryCatalog

Desire state:
Inventory doesn't know about Catalog
Catalog doesn't know about Inventory
And InventoryCatalog module is bridge between Catalog and Inventory

One more example:
Symfony core and Doctrine
Symfony doesn't know about Doctrine
Doctrine doesn't know about Symfony
And DcotrineBundle is bridge between Symfony and Doctrine

Reply:
Ok. Got it.